pkg/hq_property.py

```diff
@@ -85,19 +96,21 @@ def __init__(self, device, id, name):
 
         name -- name of the property
         """
-        self.description={'title': name, 'type': 'string', 'readOnly' : True,}#description of the property
-        super().__init__(device, id)    
-    
-    def set_RO_Value(self, propName, value: datetime):
+        super().__init__(device, id, {
+            "title": name,
+            "type": "string",
+            "readOnly": True,
+        })
+
+    def set_ro_value(self, prop_name, value: datetime):
         """
-        modifying the set_RO_Value for datetime object
+        modifying the set_ro_value for datetime object
 
         value -- value of the property, must be datetime
         """
-        #if datetime is none enter an empty date and time
+        # if datetime is none enter an empty date and time
         if value is None:
-            value = "00/00/0000\n00:00:00"
+            datetime_representation = "0000-00-00\n00:00:00"
         else:
-            value = value.strftime("%d/%m/%Y\n %H:%M:%S")#TODO:Verify if isoformat could replace strftime
-        super().set_RO_Value(propName, value)
-        
\ No newline at end of file
+            datetime_representation = value.isoformat(sep="\n", timespec="seconds")
+        super().set_ro_value(prop_name, value)
```

Comment:
value must be str

Reply:
Oops, j'ai renommé la variable mais pas partout
```python
super().set_ro_value(prop_name, datetime_representation)
```